### ISSUE TITLE
Managing dependencies on austraits upon (soft) release of `austraits 3.0.0`

### ISSUE THREAD

**Issue by fontikar** [traitecoevo/austraits#109]
Once `austraits 3.0.0` is merged into master branch, we need to update the following software to temporarily depend on the `austraits 2.2.2` (e83b53905b879fca7e826015deb67fb123741808): 

- [x] `traits.build `
- [ ] AusTraits API

Depending on 2.2.2 can only be removed once `traits.build` has reexported functions in AusTraits and tests are all passing and when ALA can download wide table can be directly from Zenodo API
